.changeset/silver-maps-grab.md

```diff
@@ -0,0 +1,56 @@
+---
+'@xstate/store': minor
+---
+
+You can now emit events from a store:
+
+```ts
+import { createStore } from '@xstate/store';
+
+const store = createStore({
+  context: {
+    count: 0
+  },
+  on: {
+    increment: (context, event, { emit }) => {
+      emit({ type: 'incremented' });
+      return { count: context.count + 1 };
+    }
+  }
+});
```

Comment:
this is a new shape of this `createStore` API, right? it would be great to mention that